### ISSUE TITLE
[Feature Request] Default value

### ISSUE THREAD

**Issue by gm0124816** [DmitryNizhebovsky/ModernControlsForAvalonia#14]
First of all, thanks for the wonderful control !

It is common that you allow people to set a range for a value, but it is also common that a default value (within that range) must be selected. Currently two controls are needed to achieve this.

Is it possible to add a third thumb that can be moved within that range changing the 'default value' ? A little different style for that thumb would be recommended.

Avalonia is getting there, the only thing missing so far is the lack of available controls, especially controls like the one you guys created ! Your effort so far is very much appreciated and keep up the good work !

**Comment by DmitryNizhebovsky** [comment]
Hello, I'm glad to hear that you liked my control!

Do I understand correctly that you would like to have a third thumb to select the default value as in this control https://www.telerik.com/products/wpf/slider.aspx ?

**Comment by gm0124816** [comment]
Yes indeed, but the default should always be within the boundaries specified by the min and max range value (it looks at first sight that that is not always the case for the telerik control).

Maybe make it on top (so it is in front of all and always visible, even if it is the same value as the min or max range), a little smaller (so the min/max range can still be handled) and preferably in a different color (why not the same color as the line in between min and max range).

I see a large number of applications for this (telerik as well :) apparently). For instance everywhere you have a 'design part' where you can set the range for that property (and the default value for that property) and a 'run-time' part where you would just have a normal slider with the min and max values as the min and max range values of your control AND the default value of your control as the suggested value of the 'normal' slider control's value.

**Comment by DmitryNizhebovsky** [comment]
Okay, this doesn't look very difficult.  I think I can implement it, but I can’t give an accurate estimate of when it will be ready because I’m busy with my main job.

**Comment by gm0124816** [comment]
Hi Dmitry,

That is fantastic ! Thanks in advance ! Take your time, your main job goes first !

You know what, why don't we leave this 'issue' open and if you are done, just put a short comment and I will be notified by email.

Once again, thanks for the effort and I'm convinced it will make your control better/easier/ergonomically usable in a number of cases !

Cheers,
Geert

**Comment by gm0124816** [comment]
Sorry for bothering again, but I noticed I made a mistake in suggesting the color for the third (value) thumb. I suggested it to be the same color as the line in between the min and max range, but since that is the same color as for the thumbs min and max range, it would not be very clear to the eye :). So I think it would be best if the color of the line outside the min and max range thumbs is used for this, resulting in a nice contrast.
